en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td>
+
+API name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>version</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API version.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>provider</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API provider name.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APILatency
+
+Represents latency values for a single API over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#latency">Latency</a>]</td>
+<td>
+
+List containing latency category values over time. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsage
+
+Represents API usage for a single timestamp. Timestamp is calculated according to the provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
+granularity value is '1d', timeSpan will represents start of the day calculated adhering to the timezone provided in
+the TimeFilter.
+Ex: timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+API usage count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageByApp
+
+Represents API usage across APIs by a single application.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>applicationId</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationName</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationOwner</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application owner name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#apiusage">APIUsage</a>]</td>
+<td>
+
+List containing the APIs usage values. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".